lib/node_modules/@stdlib/utils/is-lowercase/lib/is_lowercase.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+// MODULES //
+
+var lowercase = require( '@stdlib/string/lowercase' );
+var uppercase = require( '@stdlib/string/uppercase' );
+var isString = require( '@stdlib/utils/is-string' ).isPrimitive;
+
+
+// MAIN //
+
+/**
+* Tests if a value is a lowercase string.
+*
+* @param {*} value - value to test
+* @returns {boolean} boolean indicating whether value is a lowercase string
+*
+* @example
+* var bool = isLowercase( 'salt and light' );
+* // returns true
+* @example
+* var bool = isLowercase( 'HELLO' );
+* // returns false
+* @example
+* var bool = isLowercase( 'World' );
+* // returns false
+* @example
+* var bool = isLowercase( '!' );
+* // returns false
+*/
+function isLowercase( value ) {
+	if ( isString( value ) ) {
```

Comment:
This can be simplified to

``` javascript
return (
    isString( value )  &&
    value === lowercase( value ) &&
    value !== uppercase( value )
);
```